bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/AbstractCommunicator.java

```diff
@@ -115,40 +118,47 @@ protected void sendPacket() {
         SyncQueue syncQueue = SyncQueue.getInstance();
         IRequest request = syncQueue.peekSendQueue();
         try {
-            logger.debug("Sending packet with request={}", request);
+            logger.trace("Sending packet with request={}", request);
             byte[] packetBytes = request.getRequestPayload().getBytes();
             ParadoxUtil.printPacket("Tx Packet:", packetBytes);
             tx.write(packetBytes);
             syncQueue.moveRequest();
+        } catch (SocketException se) {
```

Comment:
```suggestion
        } catch (SocketException e) {
```

Usually we name all exceptions e